spring-cloud-stream-binder-kinesis/src/main/java/org/springframework/cloud/stream/binder/kinesis/config/KinesisBinderConfiguration.java

```diff
@@ -48,14 +53,11 @@
 	private KinesisBinderConfigurationProperties configurationProperties;
 
 	@Bean
-	public AmazonKinesisAsync amazonKinesis(
-			AWSCredentialsProvider awsCredentialsProvider,
+	public AmazonKinesisAsync amazonKinesis(AWSCredentialsProvider awsCredentialsProvider,
 			RegionProvider regionProvider) {
 
-		return AmazonKinesisAsyncClientBuilder.standard()
-				.withCredentials(awsCredentialsProvider)
-				.withRegion(regionProvider.getRegion().getName())
-				.build();
+		return AmazonKinesisAsyncClientBuilder.standard().withCredentials(awsCredentialsProvider)
+				.withRegion(regionProvider.getRegion().getName()).build();
```

Comment:
What made you to do this refactoring?
Looks like no code changes but some automatic tool from the IDE.
Would be great to switch off refactoring for code you haven't affected. 
Well, I guess the same is applied for braces in the previous class.

Just extra noise during review and these nasty comments from me...